### PR TITLE
Add Product SKU Kit support

### PR DESCRIPTION
**Pull request by gugabribeiro**
#### What is the purpose of this pull request?
When a benefit is created it is created based on the SKU's which composes the benefits itself.
The Product Kit component is now displaying the specified SKU that composes the Benefit.

#### Screenshots or example usage

<a href="https://gugabribeiro--storecomponents.myvtex.com/ninja-300/p">Click here to access the workspace</a>

captura de tela de 2018-08-31 11-59-17

captura de tela de 2018-08-31 11-59-52

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/global.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .vtex-product-kit__separator {
   width: 40px;
   height: 40px;
+  background-color: #1346d8;
```
Comment (brunojdo) [inline react/global.css]: Why? :(

File: react/global.css [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .vtex-product-kit__separator {
   width: 40px;
```
Comment (brunojdo) [inline react/global.css]: tachyons? ❤️

File: react/global.css (hunk 1 above)
Comment (andreldsa) [reply react/global.css]: 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- product SKU kit.
```
Comment (vitoriaHeliane) [inline CHANGELOG.md]: Put it into commams

Comment (vitoriaHeliane) [inline CHANGELOG.md]: Use quotations marks

File: react/global.css (hunk 2 above)
Comment (gugabribeiro) [reply react/global.css]: Tachyons has no definition for very specific sizes like this one.

File: CHANGELOG.md (hunk 3 above)
Comment (gugabribeiro) [reply CHANGELOG.md]: That's not a component it's just a new funcionality.

**Pull request merged**